tests/security/luks1/unlock_via_ssh_server.pm

```diff
@@ -45,7 +45,7 @@ sub run {
     wait_for_children;
 
     # Make sure the boot is unlocked and server should be up
-    assert_screen('displaymanager', 200);
+    assert_screen([qw(generic-desktop opensuse-welcome displaymanager)]);
```

Comment:
pls add timeout=200 back,
btw, did you add new needle here? I don't think so  :)

Reply:
Thanks for the comments,
I did not create new needle here.